lib/msal-browser/src/app/PublicClientApplication.ts

```
@@ -188,6 +193,10 @@ export class PublicClientApplication extends ClientApplication implements IPubli
             const silentCacheClient = this.createSilentCacheClient(request.correlationId);
             const silentRequest = await silentCacheClient.initializeSilentRequest(request, account);
             result = silentCacheClient.acquireToken(silentRequest).catch(async () => {
+                if (request.silentTokenRetrievalStrategy === SilentTokenRetrievalStrategy.CacheOnly) {
+                    throw new AuthError("", "Can't make network call when SilentTokenRetrievalStrategy is set to CacheOnly");
```

Please just rethrow the error thrown by `silentCacheClient.acquireToken`

---

(an additional log message would be helpful)